modules/fb_instant_articles_api_rules/fb_instant_articles_api_rules.rules.inc

```diff
@@ -0,0 +1,59 @@
+<?php
+
+/**
+ * @file
+ * Provide rules integration to manage facebook instant article using API.
+ */
+
+/**
+ * Implements hook_rules_action_info() on behalf of the Facebook Instant Articles API module.
+ */
+function fb_instant_articles_api_rules_rules_action_info() {
+  $defaults = array(
+    'parameter' => array(
+      'node' => array(
+        'type' => 'node',
+        'label' => t('Content'),
+        'description' => t('The content which needs to be managed as facebook
+          instant article.'),
+      ),
+    ),
+    'group' => t('Node'),
+    'access callback' => 'fb_instant_articles_api_rules_integration_access',
+  );
+  $actions['fb_instant_articles_api_rules_action_import'] = $defaults + array(
+    'label' => t('Add/Update facebook instant article'),
+    'base' => 'fb_instant_articles_api_rules_action_import',
+  );
+  $actions['fb_instant_articles_api_rules_action_remove'] = $defaults + array(
+    'label' => t('Remove facebook instant article'),
+    'base' => 'fb_instant_articles_api_rules_action_remove',
+  );
+  return $actions;
+}
+
+/**
+ * Facebook Instant Articles API integration access callback.
+ */
+function fb_instant_articles_api_rules_integration_access($type, $name) {
+  return user_access('administer site configuration');
```

Comment:
Let's add a new permission. Check out `fb_instant_articles_display_permission()`, `fb_instant_articles_api_permission()` etc for the convention we're using (we may as well keep it consistent). :smile: